### PR TITLE
fix: wildcard dns docs

### PR DESCRIPTION
**Pull request by spikecurtis**

### CODE REVIEW

File: docs/admin/configure.md [hunk 1]
```diff
@@ -38,8 +38,13 @@ coder server
...
 subdomain that resolves to Coder (e.g. `*.coder.example.com`).
 
-> If you are providing TLS certificates directly to the Coder server, you must use a single certificate for the
-> root and wildcard domains. Multi-certificate support [is planned](https://github.com/coder/coder/pull/4150).
+If you are providing TLS certificates directly to the Coder server, either
+
+1. use a single certificate/key for the root and wildcard domains.
+2. configure both root and wildcard domain certificates/keys via
+   [`coder.tls.secretNames`](https://github.com/coder/coder/blob/main/helm/values.yaml) in the Helm Chart, or
+   [`--tls-key-file`](../cli/server#--tls-cert-file) and [`--tls-key-file`](../cli/server#--tls-key-file) command
```
Comment (matifali) [inline docs/admin/configure.md:46]:
```suggestion
   [`--tls-cert-file`](../cli/server#--tls-cert-file) and [`--tls-key-file`](../cli/server#--tls-key-file) command
```

Comment (matifali) [reply docs/admin/configure.md:46]: @spikecurtis I guess this was a typo.

File: docs/admin/configure.md [hunk 2]
```diff
@@ -38,8 +38,13 @@ coder server
 via the dashboard or running [coder_apps](../templates/README.md#coder-apps) on an absolute path. Set this to a wildcard
 subdomain that resolves to Coder (e.g. `*.coder.example.com`).
 
-> If you are providing TLS certificates directly to the Coder server, you must use a single certificate for the
-> root and wildcard domains. Multi-certificate support [is planned](https://github.com/coder/coder/pull/4150).
+If you are providing TLS certificates directly to the Coder server, either
+
+1. use a single certificate/key for the root and wildcard domains.
+2. configure both root and wildcard domain certificates/keys via
```
Comment (deansheather) [inline docs/admin/configure.md:44]:
```suggestion
1. Use a single certificate and key for both the root and wildcard domains.
2. Configure multiple certificates and keys via
```

File: docs/admin/configure.md (hunk 1 above)
Comment (deansheather) [inline docs/admin/configure.md:46]: you should specify that this should be specified as a comma separated list or the flag can be specified multiple times

**Review by matifali — APPROVED** [review]
LGTM

**Review by deansheather — APPROVED** [review]

**Pull request merged**